tensorflow/compiler/tf2tensorrt/convert/convert_nodes.cc

```diff
@@ -5944,6 +5944,124 @@ Status ConvertTopK(OpConverterParams* params) {
   return Status::OK();
 }
 
+Status CalcDepthSpaceDynamicShape(OpConverterParams* params, int block_size,
+                                  string data_format,
+                                  nvinfer1::ITensor** first_shuffle_shape,
+                                  nvinfer1::ITensor** second_shuffle_shape) {
```

Comment:
It would be better if we use return type StatusOr<std::pair<nvinfer1::ITensor*, nvinver1::ITensor*>>.

Reply:
Changed to that. Is there a description when shall we prefer StatusOr?